noir-projects/aztec-nr/aztec/src/messages/msg_type.nr

```diff
@@ -1,3 +1,8 @@
+use protocol_types::hash::poseidon2_hash_bytes;
+
 pub global PRIVATE_NOTE_MSG_TYPE_ID: u64 = 0;
 pub global PARTIAL_NOTE_PRIVATE_MSG_TYPE_ID: u64 = 1;
 pub global PRIVATE_EVENT_MSG_TYPE_ID: u64 = 2;
+
+pub global OFFCHAIN_MESSAGE_IDENTIFIER: Field =
```

Comment:
Having this in this file feels incorrect given that it's not about individual message types but instead it's about offchain effect type. Would crate `offchain_effect_type.nr` instead. Also would add here a note that this is copied over to TS land and that it needs to be updated in TS if this changes.